selfdrive/car/toyota/values.py

```diff
@@ -1424,7 +1424,8 @@ class CAR:
   CAR.HIGHLANDERH_TSS2: dbc_dict('toyota_nodsu_hybrid_pt_generated', 'toyota_tss2_adas'),
   CAR.AVALON: dbc_dict('toyota_avalon_2017_pt_generated', 'toyota_adas'),
   CAR.RAV4_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
-  CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
+  CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated_corolla_tss2', 'toyota_tss2_adas'),
+  CAR.COROLLAH_TSS2: dbc_dict('toyota_nodsu_pt_generated_corollah_tss2', 'toyota_tss2_adas'),	', 'toyota_tss2_adas'),
```

Comment:
I think this line may have some typos in it

Reply:
Thanks i will double check

Reply:
thank you. typos fixed

Reply:
inorder to change safety param you need to modify dbc. so its not typo its more